specialists/agents/loneliness/loneliness_agent.py

```
@@ -176,16 +161,16 @@ async def initialize(self):
             logger.info("Database connections established in parallel")
             
-            # Initialize ONLY streaming Gemini client for 100% streaming experience
+            # Initialize regular Gemini client
             try:
-                self.gemini_client = get_streaming_client()
-                logger.info("Streaming Gemini client initialized (100% streaming mode)")
-            except Exception as streaming_error:
-                logger.error(f"Streaming Gemini client failed: {streaming_error}")
+                self.gemini_client = get_gemini_client()
+                logger.info("Gemini client initialized")
+            except Exception as gemini_error:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)